gnpy/core/request.py

```diff
@@ -523,10 +535,10 @@ def jsontocsv(json_data, equipment, fileout):
             # when label will be assigned destination will be with index -3, and transponder with index 2
             source = pth_el['path-properties']['path-route-objects'][0]\
                      ['path-route-object']['num-unnum-hop']['node-id']
-            destination = pth_el['path-properties']['path-route-objects'][-2]\
+            destination = pth_el['path-properties']['path-route-objects'][-3]\
```

Comment:
This is because of the order of objects
the last hop should be:
node_id , label, hop_attribute
before this PR, I did not implement the label so the hop was
node_id , hop_attribute
node_id was thus -2 in the list, 
with the pr it is -3

Reply:
Yeah, I think I understand why this had to change -- the problem I have is with this approach in general. Instead of saying something like this:
```json
"items": [
 {
    "node-id": 666,
    "frequency": 193500000,
    "power": -20.0
  }
]
```
we have this info split into two (now three) elements, one defining the `frequency`, the other `power`, etc.:
```json
"items": [
  {
    "node-id": 666,
    "frequency": 193500000
  },
  {
    "node-id": 666,
    "power": -20.0
  }
]
```
I think I understand your explanation (from Angers?) where you said this comes from the RFC draft, which itself builds upon the GMPLS on-the-wire which looks like being inspired by C-language `struct`s and linked lists. In *that* environment, making the path information modeled as a list of entities where each entity adds exactly one information to an existing node, and other types of info are added via a separate "sub packet", that makes sense.

But this is YANG, and there's no need to have such an ugly (IMHO, anyway) data structure. It's trivial to put all the required information below any node. But it's of course true that this requires a change of the YANG, and that change should be best made within the appropriate IETF WG. And because we're modifying that YANG anyway, it is (IMHO) a very good idea to really make this change.

So this was just a random complaint about the data schema, not something to address in this PR.